src/backend/src/utils/transporter.utils.ts

```diff
@@ -0,0 +1,65 @@
+import nodemailer from 'nodemailer';
+import { google } from 'googleapis';
+import SMTPTransport from 'nodemailer/lib/smtp-transport';
+import { HttpException } from './errors.utils';
+
+const { OAuth2 } = google.auth;
+
+const createTransporter = async () => {
+  try {
+    const oauth2Client = new OAuth2(
+      process.env.CLIENT_ID,
+      process.env.CLIENT_SECRET,
+      'https://developers.google.com/oauthplayground'
+    );
+
+    oauth2Client.setCredentials({
+      refresh_token: process.env.REFRESH_TOKEN
+    });
+
+    let accessToken: string | null | undefined;
+
+    await oauth2Client.getAccessToken((err, token) => {
+      if (err) {
+        console.log('*ERR: ', err);
+        throw err;
+      }
+      accessToken = token;
+    });
+
+    const transporter = nodemailer.createTransport({
+      service: 'gmail',
+      auth: {
+        type: 'OAuth2',
+        user: process.env.USER_EMAIL,
+        accessToken: accessToken?.toString(),
+        clientId: process.env.CLIENT_ID,
+        clientSecret: process.env.CLIENT_SECRET,
+        refreshToken: process.env.REFRESH_TOKEN
+      }
+    });
+    return transporter;
+  } catch (err) {
+    console.log('ERROR: ' + err);
```

Comment:
move this to line 61, it should catch any errors you're catching here and any weirdness that could occur when we actually try to call sendMail

Reply:
oops i thought this was that line my b